### ISSUE TITLE
Have autoticker recognize when a rever PR has already been submitted

### ISSUE THREAD

**Issue by scopatz** [regro/cf-scripts#102]
And skip making a secondary PR

**Comment by CJ-Wright** [comment]
I think this goes with https://github.com/regro/cf-scripts/issues/16

**Comment by viniciusdc** [comment]
@CJ-Wright  We should update [#16 ] and have this as a comment there (maybe a checklist), so we can close this.

**Comment by scopatz** [comment]
Yeah even a config variable that tells the bot to skip version bumps would be great

**Comment by viniciusdc** [comment]
> Yeah even a config variable that tells the bot to skip version bumps would be great

Maybe we could add an specific label for that PR, so bot it will recognize if the PR was already been made
